pkg/workflow_execution.go

```diff
@@ -2063,6 +2067,38 @@ func getCURLNodeTemplate(name, curlMethod, curlPath, curlBody string, inputs wfv
 	return
 }
 
+func injectFilesyncerSidecar(wf *wfv1.Workflow) error {
+	filesyncer := wfv1.UserContainer{
+		Container: corev1.Container{
+			Name:  "sys-filesyncer",
+			Image: "onepanel/filesyncer:test-server-7",
```

Comment:
I think we can update this to `v0.19.0` @Vafilor I'm creating an image for it.

Reply:
@rushtehrani Roger that.